src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/BitbucketServerVersion.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * The MIT License
+ *
+ * Copyright (c) 2016-2020, CloudBees Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.cloudbees.jenkins.plugins.bitbucket.server;
+
+import hudson.model.ModelObject;
+
+public enum BitbucketServerVersion implements ModelObject {
+    VERSION_7 {
```

Comment:
This needs to be validated it works with JCasC.

Currently we have another open issue on Webhook implementation enum.

https://github.com/jenkinsci/configuration-as-code-plugin/issues/1440

Reply:
@jetersen 
Agreed, this needs testing in general.